src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,107 @@
+
+There, depending on the path the requests is trying to access (i.e either trying to access logs or metrics) it will go through one of 2 ingresses sharing the same host : one in the loki namespace and the other in the mimir one. Note that we decided to have 2 ingresses sharing the same host instead of having 1 ingress handling all paths because usually, ingresses only forward requests in the same namespace they are deployed in. There are ways to go around this, but those are a bit tricky and definitely not straight forward.
+
+Upon reaching the correct ingress, the request will need to be authenticated before being forwarded. This is done through a nginx annotation : `nginx.ingress.kubernetes.io/auth-url` which value is the customer OIDC provider ingress' url. As the requests coming from Grafana forwards the user's OAuth identity through the use of a token, the Giant Swarm ingress only make sure that the incoming request's token matches an allowed user in the OIDC provider.
+
+Moreover, the ingressess will verify that the incoming requests are using the `X-Scope-OrgId` http headers (tenant information). If not, the request will be denied.
+
+Upon authentication, the request is then forwarded to either the `loki-gateway` or `mimir-gateway` (depending if the query is looking for logs or metrics) components which are nginx servers respectively fronting the loki and mimir clusters. Those gateway components then finally forward the request to the read components.
+
+That's it, the user now has access to the query's output.
+
+#### Write path
+
+Let's consider the following situation : a customer's logging agent running on his premises want to send its data to the Giant Swarm Observability Platform.
+
+The workflow is quite similar as the one described for the read path. The main difference is in the fact that the requests will go through another ingress in the monitoring namespace which fronts a specifically tuned alloy instance, deployed via the [alloy-gateway-app](https://github.com/giantswarm/alloy-gateway-app).
+
+Once authenticated, the request will thus go through the `alloy-gateway` which will forward it to the `loki-gateway` and then in turn the `loki-write` components.
+
+This additional step was added to ensure that we support OTLP.
+
+## Adding datasources
+
+In order for one to use the observability-platform API's read path, the most straight-forward way is to add a datasource in one's grafana pointing towards it. Here is a step-by-step guide on how to do it :
+
+* in the `Connection` section, write the observability-platform API url which always be `https://observability.<domain_name>`. Replace the `<domain_name>` placeholder with the actual domain name of your installation.
```

Comment:
I think we have some existing schemes for domain_name @pipo02mix maybe you remember?

Reply:
If not, you should be able to find it on the ingress tutorial IIRC

Reply:
I guess you mean [this page](https://docs.giantswarm.io/tutorials/connectivity/ingress/exposing-workloads/)?